packages/skuba-dive/tsconfig.json

```diff
@@ -9,6 +9,6 @@
     "removeComments": false,
     "target": "ES2020"
   },
-  "exclude": ["lib*/**/*"],
+  "exclude": ["lib*/**/*", "eslint.config.js"],
```

Comment:
Not sure if there is a nicer way. Might suss out somewhere

Reply:
<img width="541" alt="image" src="https://github.com/user-attachments/assets/6c7ec47a-90a0-454a-9bd5-f3b83fa94cde">

This also works. Am confused @AaronMoat